projects/packages/forms/src/contact-form/css/grunion.css

```diff
@@ -44,6 +44,7 @@
 	font: inherit;
 	border: 1px solid #8c8f94;
 	border-radius: 0;
+	z-index: 1;
```

Comment:
Mostly curious, but why is the `z-index` being changed? It looks like this will apply to all the different input types, but doesn't appear to be related to the select element?

Reply:
Oh, this was mainly for outlined and animated inputs so that it appears over the notched input (which is positioned absolutely)

https://github.com/user-attachments/assets/ddfdc013-896e-43f4-aa7a-0c177ca3d4ba

I can move it to `.is-style-* .wp-block-jetpack-input`